docs/topics/eap.md

```diff
@@ -38,4 +38,21 @@ check [our instructions on how to configure your build to support this version](
 
 ## Build details
 
-_No preview versions are currently available._
+<!-- _No preview versions are currently available._ -->
+
+<table>
+    <tr>
+        <th>Build info</th>
+        <th>Build highlights</th>
+    </tr>
+    <tr>
+        <td><strong>2.0.0-Beta1</strong>
+            <p>Released: <strong>June 18, 2024</strong></p>
+            <p><a href="https://github.com/JetBrains/kotlin/releases/tag/v2.0.20-Beta1" target="_blank">Release on GitHub</a></p>
+        </td>
+        <td>
+            <p>A tooling release for Kotlin 2.0.0</p>
```

Comment:
```suggestion
            <p>A tooling release for Kotlin 2.0.20</p>
```

Reply:
2.0.20 is not out yet.
all x.x.20 releases a tooling releases for the previous ones.